### PR TITLE
En 7716 below signed threshold

### PR DESCRIPTION
**Pull request by raduchis**
according to the economics document if a validator has a number of signatures included below 2.5% all the rating increases as a validator, validatorSuccess and validatorIgnored should be reverted for the current epoch.

Testing: start a testnet and shutdown a validator. During the first 2 epochs the validators rating should be decreased by the number of validator failures, while on the third epoch by the number of success and ignored

### CODE REVIEW

File: process/peer/process.go [hunk 1]
```diff
@@ -586,16 +589,22 @@ func (vs *validatorStatistics) verifySignaturesBelowSignedThreshold(
...
 
-	validatorAppereances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
-	computedThreshold := float32(validator.ValidatorSuccess) / float32(validatorAppereances)
+	validatorAppearances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
+	computedThreshold := float32(validator.ValidatorSuccess) / float32(validatorAppearances)
 
 	if computedThreshold <= signedThreshold {
-		newTempRating := vs.rater.RevertIncreaseValidator(shardId, validator.TempRating, validator.ValidatorFailure)
+		increasedRatingTimes := uint32(0)
+		if epoch < vs.belowSignedThresholdEnableEpoch {
```
Comment (iulianpascalau) [inline process/peer/process.go:601]: This is ok as it is. In other cases (where current epoch was unavailable) we used the epoch change notifier mechanism.

**Review by bogdan-rosianu — APPROVED** [review]

File: process/peer/process.go [hunk 2]
```diff
@@ -586,16 +589,22 @@ func (vs *validatorStatistics) verifySignaturesBelowSignedThreshold(
...
 	epoch uint32,
 ) error {
-
 	if epoch < vs.ratingEnableEpoch {
 		return nil
 	}
 
-	validatorAppereances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
-	computedThreshold := float32(validator.ValidatorSuccess) / float32(validatorAppereances)
+	validatorAppearances := core.MaxUint32(1, validator.ValidatorSuccess+validator.ValidatorFailure+validator.ValidatorIgnoredSignatures)
```
[the remark this answers is not in the document]
Comment (raduchis) [reply process/peer/process.go:596]: fixed

File: process/peer/process.go (hunk 1 above)
Comment (raduchis) [reply process/peer/process.go:601]: ok!

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
System tests passed.

**Pull request merged**